src/Source/C3DTilesGoogleSource.js

```diff
@@ -0,0 +1,79 @@
+import Fetcher from 'Provider/Fetcher';
+import C3DTilesSource from './C3DTilesSource';
+
+function findSessionId(tile) {
+    if (!tile) {
+        return null;
+    }
+    if (tile.content && tile.content.uri) {
+        const uri = tile.content.uri.substr(tile.content.uri.indexOf('?') + 1);
+        const sessionIds = uri.split('=');
+        return sessionIds[1];
```

Comment:
Is there a non-hacky way to do this with `URLSearchParams` and get this value with the corresponding key (i.e. `sessionId`)?

Reply:
yep, done